### PR TITLE
[US107 Task142] Create Line Charts with 3 lines

### PR DESCRIPTION
**Pull request by spark290**
[US107 Task142] Create Line Charts with 3 lines
screenshot

### CODE REVIEW

File: react-ui/src/project/index.js [hunk 1]
```diff
@@ -199,7 +199,8 @@ const Project = () => {
                         </div>
                     ) : null}
                     {selectedValue === "Burndown Chart" && isBurndown ? (
-                        <SprintDetail sprintDetails={data.sprints} attributes={data.custom_attributes} token={auth} projectName={data.name} />
+                        // <SprintDetail sprintDetails={data.sprints} attributes={data.custom_attributes} token={auth} projectName={data.name} />
```
Comment (shahih25) [inline react-ui/src/project/index.js]: Why it this commented out?

Comment (spark290) [reply react-ui/src/project/index.js]: It causes errors, I though it can be handled during integration

Comment (shahih25) [reply react-ui/src/project/index.js]: Yeah we can handle during integration. It needs to be present in div element. Also we can place it in the Sprint Detail along with the other charts.

**Review by shahih25 — APPROVED** [review]
lgtm

File: react-ui/src/project/index.js [hunk 2]
```diff
@@ -199,7 +199,8 @@ const Project = () => {
...
+                        <Graph type="Burndown Chart" />
```
Comment (shahih25) [inline react-ui/src/project/index.js]: Remove this line. We will handle it during integration. Otherwise it will cause failure.

Comment (spark290) [reply react-ui/src/project/index.js]: Did

**Review by shahih25 — APPROVED** [review]
approved

**Pull request merged**